src/components/phase/PhaseApp.js

```diff
@@ -316,9 +315,7 @@ const mapStateToProps = ({ authentication, Space }) => ({
     'geolocationEnabled',
   ]),
   userId: authentication.getIn(['user', 'id']),
-  actionEnabled:
-    authentication.getIn(['user', 'settings', 'actionEnabled']) ||
-    DEFAULT_ACTION_ENABLED,
+  actionEnabled: authentication.getIn(['user', 'settings', 'actionEnabled']),
```

Comment:
This should be `actionsEnabled` with plural `s`.

Reply:
Ah yes I thought so, but since you mentioned it without `s` I thought it was okay. I'll change it soon.